spec/amber/cli/commands/exec_spec.cr

```diff
@@ -0,0 +1,56 @@
+require "../../../spec_helper"
+require "../../../support/helpers/cli_helper"
+
+include CLIHelper
+
+module Amber::CLI
+  describe "amber exec" do
+    cleanup
+    scaffold_app(TESTING_APP)
+    `shards`
```

Comment:
Done!

Reply:
Sorry, I forgot about one of the implications of this, you have to change a line in the Dockerfile to make these tests run on Travis. Right now, the test suite is a false green because the tests for this PR are skipped.

https://github.com/sam0x17/amber/blob/amber-exec/Dockerfile#L9
Needs to be `CMD ["crystal", "spec", "-D", "run_build_tests"]`

Reply:
It has been a pending change from here: https://github.com/amberframework/amber/pull/354/files#diff-654a8a2c7db020a0d9cf8c58982c3864R9

That's the big Spec PR that has been held up for various reasons.

Reply:
You're breaking my heart man.